ios/UploadAttachments/UploadAttachments/UploadManager.swift

```diff
@@ -9,6 +9,17 @@ import UIKit
         return Singleton.instance
     }
 
+    private func buildUploadURL(baseURL: String, channelId: String, fileName: String?) -> URL? {
+        let urlComponent = NSURLComponents(string: "\(baseURL)/api/v4/files")
+        let queryChannel = URLQueryItem(name: "channel_id", value: channelId)
+        let queryFile = URLQueryItem(name: "filename", value: fileName)
+
+        urlComponent?.path = "/api/v4/files"
+        urlComponent?.queryItems = [queryChannel, queryFile]
+        print("URL TO UPLOAD \(urlComponent?.string ?? "NOT DEFINED")")
```

Comment:
Do we need to print here?

Reply:
We don’t “need” it but it can serve as it will be displayed in the logs in any case